### PR TITLE
Feature/upollo sdm 1

### PR DESCRIPTION
**Pull request by stephen-up**
### Proposed changes

Hi, I would like to add documentation for the Upollo destination for single destination mode.

Thanks

### Merge timing
- ASAP once approved?

### Related issues (optional)
Note we also have a source and a web destination.

https://segment.com/docs/connections/destinations/catalog/actions-upollo/
https://segment.com/docs/connections/sources/catalog/cloud-apps/upollo/

### CODE REVIEW

File: src/connections/destinations/catalog/upollo/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Upollo Destination
+---
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/upollo/index.md:3]:
```suggestion
---
title: Upollo Destination
id: 62fc4ed94dd68d0d189dc9b2
beta: true
---
```

File: src/connections/destinations/catalog/upollo/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+> info ""
+> The Upollo Destination is currently in beta, which means that Upollo still actively developing the destination. If you are interested in joining the beta program or have any feedback to help improve the Upollo Destination and its documentation, [let the Upollo team know](mailto:support@upollo.ai).
+
+
+## Getting started
+
+1. From the [Upollo Connections screen](https://app.upollo.ai/settings/connections), in the Segment connection, click **Configure**
+2. Under **I am not a Segment workspace admin**, click **Connect**
+3. Choose which Source should send data to the Upollo destination.
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/upollo/index.md:17]:
```suggestion
2. You will be redirected to Segment to login if you are not already
3. Choose the workspace and source from which you want to send data to Upollo.
4. Click "Authorize," and the destination will be created.
```

Changing the language based off what we display for the OAuth connection: 
Screen Shot 2023-11-22 at 9 49 54 AM

**Review by tcgilbert — CHANGES_REQUESTED** [review]
Just a couple of changes and we should be all set

File: src/connections/destinations/catalog/upollo/index.md (hunk 2 above)
Comment (stephen-up) [reply src/connections/destinations/catalog/upollo/index.md:17]: Thank you, sounds great.

File: src/connections/destinations/catalog/upollo/index.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+This destination is maintained by Upollo. For any issues with the destination, [contact Upollo's Support team](mailto:support@upollo.ai).
+
+> info ""
+> The Upollo Destination is currently in beta, which means that Upollo still actively developing the destination. If you are interested in joining the beta program or have any feedback to help improve the Upollo Destination and its documentation, [let the Upollo team know](mailto:support@upollo.ai).
+
+
+## Getting started
+
+1. From the [Upollo Connections screen](https://app.upollo.ai/settings/connections), in the Segment connection, click **Configure**
+2. You will be redirected to Segment to login if you are not already
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/upollo/index.md:18]:
```suggestion
1. From the [Upollo Connections screen](https://app.upollo.ai/settings/connections), in the Segment connection, click **Configure**.
2. You will be redirected to Segment to login if you are not already.
```

Comment (stephen-up) [reply src/connections/destinations/catalog/upollo/index.md:18]: No worries. Thanks.

File: src/connections/destinations/catalog/upollo/index.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+id: 62fc4ed94dd68d0d189dc9b2
+beta: true
+---
+
+[Upollo](https://upollo.ai?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} gives unique and actionable insights that lead to conversion, retention, and expansion.
+
+This destination is maintained by Upollo. For any issues with the destination, [contact Upollo's Support team](mailto:support@upollo.ai).
+
+> info ""
+> The Upollo Destination is currently in beta, which means that Upollo still actively developing the destination. If you are interested in joining the beta program or have any feedback to help improve the Upollo Destination and its documentation, [let the Upollo team know](mailto:support@upollo.ai).
```
Comment (forstisabella) [inline src/connections/destinations/catalog/upollo/index.md:12]:
```suggestion
> The Upollo Destination is currently in beta, which means that Upollo is still actively developing the destination. If you are interested in joining the beta program or have any feedback to help improve the Upollo Destination and its documentation, [let the Upollo team know](mailto:support@upollo.ai).
```

File: src/connections/destinations/catalog/upollo/index.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+
+## Getting started
+
+1. From the [Upollo Connections screen](https://app.upollo.ai/settings/connections), in the Segment connection, click **Configure**.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/upollo/index.md:17]:
```suggestion
1. From the [Upollo Connections screen](https://app.upollo.ai/settings/connections){:target="_blank"}, in the Segment connection, click **Configure**.
```

File: src/connections/destinations/catalog/upollo/index.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+2. You will be redirected to Segment to login if you are not already.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/upollo/index.md:18]:
```suggestion
2. Upollo redirects you to Segment. If you're not already logged in to Segment, log in now. 
```

File: src/connections/destinations/catalog/upollo/index.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+3. Choose the workspace and source from which you want to send data to Upollo.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/upollo/index.md:19]:
```suggestion
3. Select the workspace and source from which you want to send data to Upollo.
```

File: src/connections/destinations/catalog/upollo/index.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+4. Click "Authorize," and the destination will be created.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/upollo/index.md:20]:
```suggestion
4. Click **Authorize**. 
```

**Review by forstisabella — APPROVED** [review]
a few small changes and we'll be ready to ship! 🚀

File: src/connections/destinations/catalog/upollo/index.md (hunk 4 above)
Comment (stephen-up) [reply src/connections/destinations/catalog/upollo/index.md:12]: Hi @forstisabella, thanks for the feedback. Applying it now.

**Pull request merged**